hedera-platform/build.gradle.kts

```diff
@@ -23,18 +23,22 @@ val besuVersion = "23.1.2"
 val bouncycastleVersion = "1.70"
 val daggerVersion = "2.42"
 val eclipseCollectionsVersion = "10.4.0"
-val grpcVersion = "1.45.1"
-val helidonVersion = "3.0.2"
-val jacksonVersion = "2.13.3"
+val grpcVersion = "1.54.1"
+val helidonVersion = "3.2.1"
+val jacksonVersion = "2.13.5"
 val log4jVersion = "2.17.1"
 val mockitoVersion = "4.6.1"
-val nettyVersion = "4.1.77.Final"
-val protobufVersion = "3.19.4"
+val nettyVersion = "4.1.94.Final"
```

Comment:
grpc 1.54.1 requires 4.1.87.Final, this will conflict.

Reply:
@jsync-swirlds There is no conflict with `4.1.94.Final` since several other downstream components including helidon require 4.1.90.Final and 4.1.94.Final as well. We were getting a mix originally of `4.1.87.Final`, `4.1.90.Final` and `4.1.94.Final` dependencies. All tests are currently passing without modification. Under `4.1.87.Final` all the tests failed hard.

Reply:
It looks like with the netty BOM enforcing unilateral versioning at `4.1.87.Final` all tests pass without issue. We can leave this at version `4.1.87.Final`, but it does leave us possibly vulnerable to a single CVE. 

However, it is still worth merging this PR as this closes several other CVEs. One in netty is closed and at least one other in the jackson library stack.

Reply:
I didn't find any CVE in netty at the referenced version.  There is one in GRPC, but it's not relevant to our system.  The more we can keep to a single version of each dependency, the more stable the build will be (and the fewer versions to check for issues).